lib/matplotlib/collections.py

```diff
@@ -586,14 +586,15 @@ def set_linestyle(self, ls):
         """
         Set the linestyle(s) for the collection.
 
-        ===========================   =================
-        linestyle                     description
-        ===========================   =================
-        ``'-'`` or ``'solid'``        solid line
-        ``'--'`` or  ``'dashed'``     dashed line
-        ``'-.'`` or  ``'dashdot'``    dash-dotted line
-        ``':'`` or ``'dotted'``       dotted line
-        ===========================   =================
+        ==========================================  =================
```

Comment:
I guess it would make sense to only have one copy of this, rather than three (each `set_linestyle` method). No sure what the best way to do that is though. (Nor if it is worthwhile.)